doc/tutorials/transformer/model_retrieval.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<p>"For details on how to use the CLI please refer to <strong>the</strong> Transformer Extension <a href="https://github.com/exasol/transformers-extension/blob/main/doc/user_guide/user_guide.md" target="_blank">User Guide</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>